tests/lib/Qemu.py

```diff
@@ -253,6 +253,12 @@ def add_qmp(self):
         ]
         return self.monitor_file
 
+    def add_vsock(self, guest_cid):
+        self.command = self.command + [
+            '-device', 'vhost-vsock-pci,guest-cid=%d' % (guest_cid),
+        ]
+        return self.monitor_file # why are all of these returning monitor file?
```

Comment:
Did you get an answer on this? Maybe @hector-cao can chime in and we can remove the comment.

Reply:
@spmcmillan @frenchwr this return statement is useless, you can go ahead and remove it and in the function `add_qmp` function

Reply:
https://github.com/canonical/tdx/pull/216/commits/c85f2f4056d2c946edfdf53d49e14f9cdd271447